src/app/core/components/header-button/header-button.component.html

```diff
@@ -49,6 +49,14 @@
         Albums
       </a>
       <p-divider></p-divider>
+      <a
+        class="sidebar-button"
+        [routerLink]="['/settings']"
+        [routerLinkActive]="['active']"
+        [routerLinkActiveOptions]="{ exact: true }"
```

Comment:
why did you add that? This would prevent the icon be active for any subroutes. You don't have any but if you had you would like the button to be active wouldn't you? Also, because there are no nested routes why bother setting this value ?